archetypes/camel-archetype-endpointdsl/src/main/resources/archetype-resources/src/main/java/MyRouteBuilder.java

```diff
@@ -0,0 +1,46 @@
+## ------------------------------------------------------------------------
+## Licensed to the Apache Software Foundation (ASF) under one or more
+## contributor license agreements.  See the NOTICE file distributed with
+## this work for additional information regarding copyright ownership.
+## The ASF licenses this file to You under the Apache License, Version 2.0
+## (the "License"); you may not use this file except in compliance with
+## the License.  You may obtain a copy of the License at
+##
+## http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## ------------------------------------------------------------------------
+package ${package};
+
+import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.builder.endpoint.dsl.FileEndpointBuilderFactory;
+
+import static org.apache.camel.builder.endpoint.StaticEndpointBuilders.file;
+import static org.apache.camel.builder.endpoint.StaticEndpointBuilders.timer;
+/**
+ * A Camel Java DSL Router
+ */
+public class MyRouteBuilder extends RouteBuilder {
```

Comment:
Its better to extend EndpointRouteBuilder, then you dont need those static imports, see for example https://github.com/apache/camel/blob/4ea9e6c357371682b855d2d79655b41120331b7a/core/camel-endpointdsl/src/test/java/org/apache/camel/builder/endpoint/FileDelayTest.java#L50

Reply:
Commit [77cd11e](https://github.com/apache/camel/pull/4542/commits/77cd11e9e2954d9723499329159ab5b2ff042825) resolves both observations. Thank you for your support.